modin/pandas/dataframe.py

```diff
@@ -448,11 +448,7 @@ def groupby(
         elif hashable(by) and not isinstance(by, pandas.Grouper):
             drop = by in self.columns
             idx_name = by
-            if (
-                self._query_compiler.has_multiindex(axis=axis)
```

Comment:
why this check is removed?

Reply:
Because this restriction didn't allow to groupby by index level in case index didn't have multiple levels